dev/group_vars/sp.yml

```diff
@@ -0,0 +1,80 @@
+# ----------------------------------------------------------------------------
+#  Copyright (c) 2018 WSO2, Inc. http://www.wso2.org
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+# ----------------------------------------------------------------------------
+
+wso2_group: wso2
+wso2_user: wso2carbon
+setup_path: /usr/lib/wso2
+product_name: wso2sp
+product_version: 4.2.0
+product_path: /opt/wso2
+product_archive_deb: wso2sp-linux-installer-x64-4.2.0.deb
+product_archive_rpm: wso2sp-linux-installer-x64-4.2.0.rpm
+product_package_location: files
+jdbc_driver: mysql-connector-java-5.1.45-bin.jar
+systemd_path: /etc/systemd/system
```

Comment:
Do we need to parameterize this since this is not going to change from OS to OS? 
WDYT?